core-rust/state-manager/src/metrics.rs

```diff
@@ -114,6 +115,14 @@ impl StateManagerMetrics {
                 &["source", "rejection_reason"],
             )
             .unwrap(),
+            mempool_from_local_api_to_commit_wait: new_timer(
+                opts(
+                    "mempool_from_local_api_to_commit_wait",
+                    "Time spent in the mempool, by a transaction coming from local API, until successful commit."
+                ),
+                vec![0.01, 0.1, 0.5, 1.0, 2.5, 5.0, 10.0, 25.0, 50.0]
```

Comment:
_(Note from author)_

On java side, we actually report all timings using only the basic `Summary` (no histograms), but here I hope to really get some insight from buckets, so let's have them at least temporarily (despite the cost).